docs/choreolib/auto-routines.md

```diff
@@ -1,7 +1,92 @@
 # Auto Routines
 
+## Setup
+
+### AutoFactory
+
 ChoreoLib provides the `AutoFactory` class as higher level API to make it easier to create competitive and complex auto routines inside your robot code.
```

Comment:
```suggestion
ChoreoLib provides the `AutoFactory` class as a higher level API to make it easier to create complex auto routines inside your robot code.
```

Reply:
Why not include competitive?

Reply:
That's a quality that we can't guarantee from the Choreo side. If competitiveness equates to complexity, it's redundant anyway.